### PR TITLE
feat: display all child tags in the "bare bones" taxonomy detail page [FAL-3529] [FC-0036]

### PR DESCRIPTION
**Pull request by bradenmacdonald**
This is a minor improvement to https://github.com/openedx/frontend-app-course-authoring/pull/655

It sort of depends on https://github.com/openedx/openedx-learning/pull/119 but it can be merged even before that one is merged.

This is **not** the final UI design and all of this can/will be changed - this just makes sure that users can see _all_ the tags in the taxonomy during our initial round of testing.

## Before:

Screenshot 2023-11-17 at 3 22 16 PM

## After:

Screenshot 2023-11-17 at 3 23 01 PM

Screenshot 2023-11-17 at 3 23 16 PM

Screenshot 2023-11-17 at 3 23 30 PM

### CODE REVIEW

**Comment by bradenmacdonald** [comment]
@pomegranited @rpenido FYI - I don't want to delay https://github.com/openedx/frontend-app-course-authoring/pull/655 so I've made this as a separate PR, but this is an improvement I want to make before we update our sandbox and get users testing it.

**Comment by rpenido** [comment]
Hi @bradenmacdonald!

LGTM. Just need to fix some lint issues to make the tests pass.

If running without https://github.com/openedx/openedx-learning/pull/119 there is some weird behaviors:
Loading the taxonomy imported using our template file it shows the children (all depths) alongside the parents.
image

Edit: ~I didn't dig too much into it, so the problem is in the backend but I don't know if it is in the `get_filtered_tags` function, the import code or the template itself.~

~Should we create a task to take a look at this?~

Running the https://github.com/openedx/openedx-learning/pull/119 branch fixed it.

File: src/generic/Loading.jsx
Comment (rpenido) [inline src/generic/Loading.jsx]: Fix lint
```suggestion
```

File: src/taxonomy/tag-list/TagListTable.jsx
Comment (rpenido) [inline src/taxonomy/tag-list/TagListTable.jsx]: lint 
```suggestion
```

File: src/taxonomy/tag-list/TagListTable.jsx
Comment (rpenido) [inline src/taxonomy/tag-list/TagListTable.jsx]: lint 
```suggestion
  }
  
  if (subTagsData.isError) {
```

**Review by rpenido — APPROVED** [review]
LGTM :+1: 
* [x]  I read through the code
* [x]  I followed the test instructions and ran the code locally
* [x]  I checked for accessibility issues by navigating using the keyboard only
* [x] User-facing strings are extracted for translation

@bradenmacdonald 
Do you think it is better to merge in the branch opencraft branch now or wait https://github.com/openedx/frontend-app-course-authoring/pull/655 get merged and then create a new PR against `master`?

**Comment by bradenmacdonald** [comment]
@rpenido Thanks! I'll wait until your PR is merged (as well as https://github.com/openedx/openedx-learning/pull/119 ), and then I'll re-submit this PR. I don't want to slow down your PR from getting merged.

**Comment by bradenmacdonald** [comment]
@rpenido Now that your original PR has merged, I have a new PR. Can you please review it there? https://github.com/openedx/frontend-app-course-authoring/pull/703

**Pull request closed without merging**
